services/MerchStoreService.ts

```diff
@@ -109,6 +149,79 @@ export default class MerchStoreService {
     });
   }
 
+  /**
+ * Verify that collections have valid photots.
+ */
+  private static verifyCollectionHasValidPhotos(collection: MerchCollection | MerchandiseCollectionModel) {
+    if (collection.collectionPhotos.length > Config.file.MAX_COLLECTION_PHOTO_COUNT) {
```

Comment:
Lets move the constant to this file similar to here: https://github.com/acmucsd/membership-portal/pull/351/files#diff-8f5e873f32fd041a327e36c028853a65ced08b99cb09572d3d785ef193b0c212R42